src/components/customizableAside.module.css

```diff
@@ -0,0 +1,25 @@
+/* customizableAside css */
+
+    .asideContainer {
```

Comment:
`kebab-case` for CSS properties please!

Reply:
ok ma

Reply:
@Sheyitofunmi It's alright dear. continue with camelCase